store/storage/sqlite/db.go

```diff
@@ -209,6 +214,40 @@ func (db *Database) ReverseIterator(storeKey string, version uint64, start, end
 	return newIterator(db.storage, storeKey, version, start, end, true)
 }
 
+// Restore implements the StorageSnapshotter interface.
+func (db *Database) Restore(version uint64, chStorage <-chan *store.KVPair) error {
```

Comment:
LGTM. But we'll need to do this for all SS backends. Whats your plan here? To abstract this or to implement for RocksDB and PebbleDB later?

Reply:
I will create the abstraction layer for all backends later